tests/input/html/html_no_extension

```diff
@@ -0,0 +1,1487 @@
+<!DOCTYPE html>
```

Comment:
Be really careful with the sample html file you provide. It should not contain sensitive data.
I suggest you to choose something simpler.
Note: i will probably squash yours commits to have one single commit related to your fix as soon as your PR is validated, this will avoid get all the intermediate commit in the develop branch history.

Reply:
Sorry about the issues on my commits. I'm trying to be better. Thanks for the patiency.

Reply:
Data changed. Thanks for the tip